Git-commit: 9b1cc41774b9d6016f41ff2fa0901df9df72adfb
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 12 Feb 2021 17:45:57 +0000
Subject: [PATCH] Partial revert "bcm2711: Disable bsc_intr and aon_intr by
 default and enable in overlay"
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

raspberrypi inclusion
category: feature
bugzilla: 50432

--------------------------------

This reverts commit c765fd45856a34b9e46daa9263faeafe006c3985.

bsc_intr is being removed again as it causes issues.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
Signed-off-by: Fang Yafen <yafen@iscas.ac.cn>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm/boot/dts/bcm2711.dtsi                         | 1 -
 arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts | 7 -------
 2 files changed, 8 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711.dtsi b/arch/arm/boot/dts/bcm2711.dtsi
index 9d2730c27637..be2f9ba08ea8 100644
--- a/arch/arm/boot/dts/bcm2711.dtsi
+++ b/arch/arm/boot/dts/bcm2711.dtsi
@@ -314,7 +314,6 @@ bsc_intr: interrupt-controller@7ef00040 {
 			interrupts = <GIC_SPI 117 IRQ_TYPE_LEVEL_HIGH>;
 			interrupt-controller;
 			#interrupt-cells = <1>;
-			status = "disabled";
 		};
 
 		aon_intr: interrupt-controller@7ef00100 {
diff --git a/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts
index 8e9220f303f0..6a11260a4f1a 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-pi4-overlay.dts
@@ -167,13 +167,6 @@ __dormant__  {
 	};
 
 	fragment@23 {
-		target = <&bsc_intr>;
-		__overlay__  {
-			status = "okay";
-		};
-	};
-
-	fragment@24 {
 		target = <&aon_intr>;
 		__overlay__  {
 			status = "okay";
-- 
2.31.1

